remove references to "Silex" (#40685)
symfony/symfony: src/Symfony/Component/HttpKernel/README.md

The HttpKernel component provides a structured process for converting a Request
into a Response by making use of the EventDispatcher component. It's flexible

> enough to create a full-stack framework (Symfony), a micro-framework (Symfony) or
> an advanced CMS system (Drupal).

Reviewer: ```suggestion
enough to create full-stack frameworks, micro-frameworks or advanced CMS system like Drupal.
```

The HttpKernel component provides a structured process for converting a Request
into a Response by making use of the EventDispatcher component. It's flexible

> enough to create full-stack frameworks, micro-frameworks or advanced CMS system like Drupal.

Reviewer 2: `systems` should also be plural, like previous items in the list.
